playwright.config.ts

```diff
@@ -7,7 +7,7 @@ export default defineConfig({
   testDir: './tests/e2e',
   /* Run tests in files in parallel */
   fullyParallel: true,
-  workers: 4,
+  workers: 1,
```

Comment:
This is a head scratcher - I initially set this to 1 (and retries to 0) to make it easier to debug things when they run serially, but what I found is that with this config I don't see any test being flaky at all and I did restart e2e tests 9 times to get 10 runs with this config and they all 100% pass without any retries ...

The problematic bit is that I don't quite understand why this would unflake things, but it seems to do it?

I also don't see duration increase with this change - in fact I see decrease with github actions consistently finishing in somewhere between 4-5 minutes, while with 4 workers it often seems to take 5+ minutes (likely due to retries and flakiness which cause redeploys)

Reply:
🤷🏼‍♂️ Not very satisfying, but I'll take it. Thanks for digging!